platform/build-faiss/build.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+if [ -d /origin ]; then
+  cd /origin/platform/build-faiss
+else
+  echo "artificialwisdomai/origin project needs to exist"
+  exit 1
+fi
+
+if [[ ! `id -u` -eq 0 ]]; then
+  echo "This needs to run as root"
+  exit 1
+fi
+
+export PATH=$HOME/.local/bin:$PATH
+export DEBIAN_FRONTEND=noninteractive
+
+apt-get update && apt-get dist-upgrade -y
+
+# Install python and build essentials and essential libraries
+apt-get install -y python3-venv python3-pip python3-dev build-essential libssl-dev libffi-dev libxml2-dev libxslt1-dev liblzma-dev libsqlite3-dev libreadline-dev libbz2-dev neovim curl git wget
+
+# Update Setuptools
+python3 -m pip install -U pip setuptools wheel
+
+# Add a couple Python prerequisites
+pip install numpy swig torch
+
+# Get Intel OneAPI for BLAS support
+# From: https://www.intel.com/content/www/us/en/docs/oneapi/installation-guide-linux/2023-0/apt.html
+
+# download the key to system keyring
+wget -O- https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB \
+| gpg --dearmor | tee /usr/share/keyrings/oneapi-archive-keyring.gpg > /dev/null
+
+# add signed entry to apt sources and configure the APT client to use Intel repository:
+echo "deb [signed-by=/usr/share/keyrings/oneapi-archive-keyring.gpg] https://apt.repos.intel.com/oneapi all main" | tee /etc/apt/sources.list.d/oneAPI.list
+
+apt update
+apt install dkms intel-basekit -y
+
+## Get CUDA and install it
+
+curl -sLO https://developer.download.nvidia.com/compute/cuda/12.2.0/local_installers/cuda_12.2.0_535.54.03_linux.run
```

Comment:
why not use the apt packaging? Sure, the run packaging works, was this just testing?